server/src/services/domain/localization.service.ts

```diff
@@ -0,0 +1,24 @@
+import * as i18n from 'i18n';
+
+export const getLocalizedMessage = (
```

Comment:
It's actually difficult to understand. What does it mean here when we say we want to get localized message not only by name but by other parameters,  @danbilokha  could you please bring up an example

Reply:
added example